core/unit_test/TestViewLayoutStrideAssignment.hpp

```diff
@@ -888,4 +888,6 @@ TEST(TEST_CATEGORY_DEATH, view_layoutstride_left_to_layoutright_assignment) {
 
 }  // namespace Test
 
+#ifdef KOKKOS_ENABLE_IMPL_VIEW_LEGACY
 #include <TestIrregularLayout.hpp>
+#endif
```

Comment:
Maybe, add a FIXME so the test can be converted later?

Reply:
Sure, this just needs a completely new way (i.e. mdspan style) to do custom layouts.